spec/acceptance/resources_controller_spec.rb

```diff
@@ -42,6 +42,21 @@
       expect(status).to be(200)
       expect(JSON.parse(response_body)["included"].count).to be(9)
     end
+
+    it "only get name and system of resources" do
+      do_request 'fields[resource]': "name,system"
+
+      expect(status).to be(200)
+      data = JSON.parse(response_body)["data"][1]
+      attrs = data["attributes"]
+      expect(attrs.size).to be(1)
+      expect(attrs["name"]).not_to be_nil
+      expect(attrs["description"]).to be_nil
+      relationships = data["relationships"]
+      expect(relationships.size).to be(1)
+      expect(relationships["system"]).not_to be_nil
+      expect(relationships["translations"]).to be_nil
```

Comment:
if you're just checking keys, you could do this:
```suggestion
      expect(attrs.keys).to eq ["name"]
      relationships = data["relationships"]
      expect(relationships.keys).to eq ["system"]
```

Reply:
thanks, thats much more succinct, going to make one more adjustment